src/app/tab-preferences/tab-preferences.page.ts

```diff
@@ -81,6 +82,8 @@ export class Tab3Page {
 
   premiumLabel: string = ""
 
+  package = ""
```

Comment:
Could be private.

Reply:
Private fields can not be accessed in the corresponding template. But you are right public is excessive, will change to protected

Reply:
mb, I only used "Find All References" and VSCode is not able to pick up the use in templates.